pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -230,7 +197,8 @@ def execute(
             TensorLike, tuple[TensorLike], tuple[tuple[TensorLike]]: A numeric result of the computation.
         """
 
-        return self._interface.execute(circuits, execution_config)
+        # TODO: remove comment when cuTensorNet MPS backend is available as a prototype
+        # return self._interface.execute(circuits, execution_config)
```

Comment:
Same question as above?

Reply:
Same as above, but with `execute` instead of `preprocess`